server/src/routes/api/v1/characterRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express"
+
+import {Character} from "../../../models/index.js"
```

Comment:
There should be spaces around `Character` in the destructuring
```js
{ Character }
```
https://github.com/airbnb/javascript#whitespace--in-braces